src/ds/prob/count-min-sketch.md

```diff
@@ -0,0 +1,38 @@
+In this tutorial you will learn how to use a count-min sketch to track profits in a bike shop use case.
+
+A count-min sketch is used to determine the frequency of events in a stream. You can query the count-min sketch to get an estimate of the frequency of any given event.
+
+You can initialize the count-min sketch and specify a width (the number of counters in each array) and depth (the number of counter-arrays).
```

Comment:
These parameters mean nothing to the novice user. We should refer here to an external source that explains them.

Reply:
Please provide an external reference.